### PR TITLE
Capstone Metrics Webapp (Mini World Atlas)

### PR DESCRIPTION
**Pull request by katarighe**
In **this capstone project**, I have created a React mobile web application called _Mini World Atlas_, which allows users to check a list of metrics (numeric values) retrieved from a selected API called REST countries, where you can get information about countries via a RESTful API.

**REST Countries API** is a RESTful API that provides information about 200+ countries and territories. It is a free and open-source API, and it does not require an API key. The API is based on the JSON (JavaScript Object Notation) format, and it provides information about a country's name, flag, coat of arms, capital, population, area, borders, currencies, languages, and more.

Here are **my main accomplishments** done for this capstone project:

- I **selected** the API that would be used, called [REST Countries](https://restcountries.com/).
- I **built** the file structure for the React app, such as `components`, `assets`, `tests`, and `redux`. 
- I **prepared** the routes and navigation in the app, such as `index.js` and `App.js`. 
- I **ensured** that a user could display a list of items and filter them, with data coming from the _REST API_.
- I **created** tests for the application, as well as the snapshots using _Jest_ and _React Testing Library_. 
- I **styled** the components to match the provided design.
- I **deployed** the project and tested it for final details using _Netifly_. 
- I **recorded** a video for the project presentation using _Loom_. 
- I **created** a good README and PR description detailing the capstone project.

Overall, I am pleased with the work that I have accomplished using React, Redux, JSX, CSS, and JavaScript. I therefore believe that the use of the correct syntax, documentation, components, events, store, actions, reducers, props, routers, styles, modularization, callbacks and promises, Babel, Linters, Webpack, React best practices, component & integration tests, and the Gitflow workflow, has resulted in a high-quality, maintainable, and reliable React webapp. 

> Please visit these links in order to see a hands on of my app.

**📹 Video Presentation:** 
https://www.loom.com/share/5aa00e1311db43879505ce828a033304

**🌎 Live Website:**
https://benevolent-crostata-5c4bea.netlify.app/

_Due to the fact that it is a web app that is not optimized for desktop use, the live demo link is best viewed in mobile mode, please see the [README file](https://github.com/katarighe/capstone-metrics-webapp/blob/development/README.md) for instructions on how to optimize this website into mobile mode.._

**🔍 Preview:**

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,214 @@
...
+Give a star⭐️ or a thumbs up 👍 if you like this project! You can visit my GitHub profile for more of my projects.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Project supervised by [Microverse](https//www.microverse.org/)
+- [Creative Commons 4.0 License](https://creativecommons.org/licenses/by-nc/4.0/)
```
Comment (TedLivist) [inline README.md:203]: - [ ] Your credit should be ascribed to Nelson Sakwa directly who inspired the project design, not just Creative Commons License.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @katarighe,

Good job so far!
✔️ Implementation of search functionality
✔️ Well-written and passing tests
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (katarighe) [reply README.md:203]: ✅ Done

**Review by gemmen29 — APPROVED** [review]
Hi @katarighe, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**